scrapers/move_scraper.py

```diff
@@ -70,7 +73,9 @@ def _scrape_move(self, move_url: str) -> Optional[MoveInfo]:
                 healing=(move_data["meta"]["healing"] / 100),
                 drain=(move_data["meta"]["drain"] / 100),
                 hit_info=hit_count_info,
-                accuracy=(move_data["accuracy"] / 100 if move_data["accuracy"] else None)
+                accuracy=(move_data["accuracy"] / 100 if move_data["accuracy"] else None),
+                ailment=Ailment(ailment.upper()) if ailment not in self.ailment_blacklist else None,
+                ailment_chance=move_data["meta"]["ailment_chance"] / 100,
```

Comment:
I would check to see if ailment_chance is 0 here (in API result) and map that to a 100% chance, if you're checking for it later.